src/components/Reports/SubscriberHistory/SubscriberHistoryCurrentSearch/SubscriberHistorySentCampaigns/SubscriberHistorySentCampaigns.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useEffect } from 'react';
+import { InjectAppServices } from '../../../../../services/pure-di';
+import { Loading } from '../../../../Loading/Loading';
+import { FormattedMessage } from 'react-intl';
+
+const SubscriberHistorySentCampaigns = ({ subscriber, dependencies: { dopplerApiClient } }) => {
+  const [state, setState] = useState({ loading: true });
```

Comment:
Why not set state for loading inside fetch as in the previous component? This way all side effects are inside `useEffect`

Reply:
Yes but need before too because when render the view before enter effect for a second it's broken